Bottom: 3dd2f13c64af60dd31e6a26b267360c861471b5a
Top:    6c408210c0298e653561feaf37a2d13c87d5c671
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-25 17:47:17 +0100

Refresh of draw-c-split-gathering-routine

---

diff --git a/src/draw.c b/src/draw.c
index 13c348b..2cfaefa 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -534,7 +534,7 @@ pin_callback (const BoxType * b, void *cl)
 }
 
 static void
-_draw_pad_name (PadType *pad)
+draw_pad_name (PadType *pad)
 {
   BoxType box;
   bool vert;
